articlequality/feature_lists/ptwiki.py

```diff
@@ -0,0 +1,169 @@
+"""
+Portuguese Wikipedia
+++++++++++++++++++++
+"""
+import re
+
+from revscoring import Feature
+from revscoring.datasources.meta import filters, mappers
+from revscoring.features import wikitext
+from revscoring.features.meta import aggregators
+from revscoring.features.modifiers import log, max, sub
+from revscoring.languages import portuguese
+
+from . import wikipedia
+
+# Templates
+infobox_templates = wikitext.revision.template_names_matching(
+    r"Info", name="ptwiki.revision.infobox_templates")
+CN_TEMPLATES = [
+    r"Carece de fontes",
+    r"Carece de fontes2",
+    r"Carece de fontes/bloco",
+    r"Carece de fontes/bloco2"
+]
+cn_templates = wikitext.revision.template_names_matching(
+    "|".join(CN_TEMPLATES), name="ptwiki.revision.cn_templates")
+MAIN_TEMPLATES = [
+    r"Artigo[ _]principal",
+    r"Ver[ _]artigo[ _]principali",
```

Comment:
Did you mean `r"Ver[ _]artigo[ _]principal"`? (withouth the extra "i" at the end)